src/fipp/visit.cljc

```diff
@@ -1,5 +1,6 @@
 (ns fipp.visit
   "Convert to and visit edn structures."
+  (:refer-clojure :exclude [boolean?])
```

Comment:
Have you tested this on CLJS? I'm not sure if multiple `:refer-clojure` forms is valid. Even if it works, I think I'd prefer to put the feature expression on the individual entry in the vector, rather than have duplicate clauses.